src/wasm-lib/kcl/src/std/sketch.rs

```diff
@@ -1115,20 +1118,103 @@ async fn make_sketch_plane_from_orientation(
 ) -> Result<Box<Plane>, KclError> {
     let mut plane = Plane::from_plane_data(data.clone(), exec_state);
 
-    // Get the default planes.
-    let default_planes = args
-        .ctx
-        .engine
-        .default_planes(&mut exec_state.id_generator, args.source_range)
-        .await?;
-
     plane.id = match data {
-        PlaneData::XY => default_planes.xy,
-        PlaneData::NegXY => default_planes.neg_xy,
-        PlaneData::XZ => default_planes.xz,
-        PlaneData::NegXZ => default_planes.neg_xz,
-        PlaneData::YZ => default_planes.yz,
-        PlaneData::NegYZ => default_planes.neg_yz,
+        PlaneData::XY => {
+            let id = exec_state.id_generator.next_uuid();
+            args.batch_modeling_cmd(
+                id,
+                ModelingCmd::from(mcmd::MakePlane {
+                    clobber: false,
+                    origin: KPoint3D { x: LengthUnit(0.0), y: LengthUnit(0.0), z: LengthUnit(0.0) },
+                    size: LengthUnit(60.0),
+                    x_axis: KPoint3D2 { x: 1.0, y: 0.0, z: 0.0 },
```

Comment:
We have some convenience methods that can help here, e.g. [this](https://docs.rs/kittycad-modeling-cmds/latest/kittycad_modeling_cmds/shared/struct.Point3d.html#method.only_x)
```suggestion
                    x_axis: KPoint3D2::only_x(1.0),
```